### PR TITLE
KFSPTS-6098: refactored code to account for copied DVs bug fix contri…

### PR DESCRIPTION
**Pull request by bhutchinson**
…bution

### CODE REVIEW

**Comment by bhutchinson** [comment]
@jhulslander There is still one failing unit test in the PR suite that I believe is from a missing property in the config properties file on Jenkins for the upgrade PR suite. I will hopefully be able to fix that soon, but in the meantime, I think this is good to review and merge if you think it looks ok.

File: src/main/java/edu/cornell/kfs/sys/CUKFSKeyConstants.java [hunk 1]
```diff
@@ -165,8 +165,6 @@
     public static final String ERROR_CSACCOUNT_CONTINUATION_ACCOUNT_CLOSED = "error.csAccount.continuationAccount.closed";
     public static final String WARNING_CSACCOUNT_CONTINUATION_ACCOUNT_USED = "warning.csAccount.continuationAccount.used";
 
-    public static final String MESSAGE_DV_PAYEE_INVALID_PAYMENT_TYPE = "message.dv.payee.invalid.payment.type";
-
-    // KFSPTS-4792   
+    // KFSPTS-4792
```
Comment (jhulslander) [inline src/main/java/edu/cornell/kfs/sys/CUKFSKeyConstants.java]: do we need this comment?

Comment (bhutchinson) [inline src/main/java/edu/cornell/kfs/sys/CUKFSKeyConstants.java]: probably not, but there are a bunch of these unrelated to our contribution changes so I left them

**Comment by jhulslander** [comment]
OK, this looks fine .... sorry I missed the PR before.

**Comment by bhutchinson** [comment]
Thanks Jay - you didn't miss it, I was waiting to assign it to you until it was ready.

**Pull request merged**